cdip_admin/integrations/models/v2/models.py

```diff
@@ -192,6 +192,18 @@ def _pre_save(self, *args, **kwargs):
             if "broker" not in self.additional:
                 self.additional.update({"broker": "gcp_pubsub"})
 
+        if self.is_er_site:
+            # Cleanup
+            url_parse = urlparse(self.base_url)
+            if url_parse.scheme == "http":
+                url_parse.scheme = "https"
```

Comment:
I'm not confident you'll be able to set this. Can you try this out locally?

Reply:
agree, please run unit tests locally, and add some new ones to cover this case if you have time

Reply:
also, can we fix the commits history to include only the commits that are really from this PR?

Reply:
I am not sure why all those commits were included, but let me check